docs/cruiser/walkthrough/Project Setup.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 2
+---
+
+# Project Setup
+
+## Prerequisites
+
+First we need to ensure we have the needed prerequisites installed.
+
+### Rust
+
+[Install Rust](https://www.rust-lang.org/tools/install).
+
+### Solana
+
+[Install Solana](https://docs.solana.com/cli/install-solana-cli-tools). Cruiser currently only supports rust `1.59` which exists on solana versions `>=1.9.17`.
+
+## Project Initialization
+
+Now we'll set up our first cruiser project. We'll start by running `cargo init cruiser_tutorial --lib` to create and init a new folder for our project. 
+This comes with a few default files, some of which we'll overwrite. Eventually our project structure will look like this. The `*` marks files that we end up with after this step.
+
+```
+cruiser_tutorial
+*|- src
+ | |- accounts           // Accounts used by your program and their impls
+ | | |- mod.rs
+ | | |- account1.rs
+ | | |- account2.rs
+ | | |- ...
+ | |- arguments          // `AccountArgument`s used by your program
+ | | |- mod.rs
+ | | |- argument1.rs
+ | | |- argument2.rs
+ | | |- ...
+ | |- instructions       // `Instruction`s used by your program
+ | | |- mod.rs
+ | | |- instruction1.rs
+ | | |- instruction2.rs
+ | | |- ...
+*| |- lib.rs             // Your program's main file
+ | |- pda.rs             // Your program's PDAs
+ |- tests
+ | |- all_tests.rs       // Tests for your program
+ | |- instructions
+ | | |- mod.rs
+ | | |- test_instruction1.rs
+ | | |- test_instruction2.rs
+ | | |- ...
+*|- Cargo.toml
+*|- Cargo.lock
+*|- rust-toolchain.toml
+```
+
+### `rust-toolchain.toml`
+
+We need to add `rust-toolchain.toml`, as it will keep Rust relatively in-line with the current solana version.
+
+```toml
+[toolchain]
+channel = "nightly-2022-03-22"
+components = ["rustfmt", "clippy"]
+profile = "minimal"
+```
+
+### `Cargo.toml`
+
+The `Cargo.toml` file will add features and the `cruiser` dependency.
+
+```toml
+[package]
+name = "tutorial_program"
```

Comment:
this does not match the name of previous cargo init name. `cruiser_tutorial`

Reply:
Technically it doesn't need to but it should